src/main/java/mdlaf/components/tabbedpane/BasicTabbedPaneUIAdapter.java

```diff
@@ -527,9 +526,9 @@ protected void paintTab(Graphics g, int tabPlacement,
                     tabScroller.croppedEdge.getTabIndex() == tabIndex && isHorizontalTabPlacement()) {
                 int availTextWidth = tabScroller.croppedEdge.getCropline() -
                         (textRect.x - tabRect.x) - tabScroller.croppedEdge.getCroppedSideWidth();
-                clippedTitle = SwingUtilities2.clipStringIfNecessary(null, metrics, title, availTextWidth);
+                clippedTitle = BasicGraphicsUtils.getClippedString(tabPane, metrics, title, availTextWidth);
```

Comment:
Please restore the SwingUtilities2 also in this class, I can not build your branch with the JDK8

Reply:
@mikera, sorry you can restore the change inside this class or you can remove this call.

With this code the library not compiling with JDK8.

Thanks